### ISSUE TITLE
Need clear setup instruction

### ISSUE THREAD

**Issue by rasakahmed** [KaliyaAlkaline/InteractionAPI#1]
Well I'm little dumb to understand your setup process. Can you help me regarding this?

**Comment by KaliyaAlkaline** [comment]
Yeah, sorry if it's a bit confusing. This code comes from last year, so it could probably use some revision on my end. It uses a a couple depreciated libraries that make it inefficient, so I may just remake it.

If you want to work with what code is up now, here is a step-by-step:

**For hosting:**
1. Make sure you have npm and NodeJS installed on your system.
2. Create a new folder (e.g: "Test") and navigate to it via whatever command line tool you have (on Windows10 you would write `cd Test` in Command Prompt).
3. Use the command: `npm i express cors body-parser xmlhttprequest nodemon`
4. The `package.json` file should already be downloaded, so type `npm start` to start the server.

**For using:**
1. Copy the code from `clientside.js` and paste it in an environment of your choosing. This code can be ran from any browser, but make sure if you are using a browser that it's executed on a page without CORS rules (I like using `about:blank`).
2. Go to Discord, open dev tools (ctrl+shift+i on Windows), then navigate to the tab in the docker that says "Network".
3. Make sure you type "API" into the filter section of the Network tab, then use the slash command you wish to bot.
4. After executing the slash command, there should be a new object that appears in the space below the filter titled "interactions". Click on it and a mini docker will appear; navigate to "payload". Once you're in the payload section, copy the text below (excluding "payload_json:").
5. Now, with the slash data copied, you can navigate back to whichever environment has the `clientside.js` code and execute the function `InteractionsAPI.post()` with the copied data as the value in parentheses. This function should return a randomly generated key.
6. Now, to execute the slash command programmatically, simply use the function `InteractionsAPI.request(<your token>, <target channel ID>, <randomly generated key>)`, replacing the objects in `<>` with their respective values of course.

If this seems complicated, that's because it is. I had to look back through my own code just to answer this, and it's loaded with redundancy. I never went back to this project since no one has had any need for the tool, but now I feel like fixing it.
